src/driver/driver.cpp

```diff
@@ -187,9 +187,12 @@ SolverOptions buildSolverOptions(const json& case_data)
 		}
 
 		if (case_data["solver_options"].contains("high_order_mesh")) {
-			res.meshTypeCurved = true;
+			res.curvedOperator = true;
```

Comment:
I do not have a feeling on what this switch does by its name. Can we think of something more self-explanatory?